lib/options.js

```diff
@@ -173,13 +174,15 @@ exports.parse = function parse(argv) {
   } else {
     throw new ExitError(fmt('Invalid cluster option: %s', cluster));
   }
-
-  if (options.channel && url.parse(options.channel).protocol) {
+  debug(options.channel);
+  var protocol;
+  if (options.channel && (protocol = url.parse(options.channel).protocol)) {
     options.channel = defaults(options.channel, {
       path: 'supervisor-control'
     }, {
-      protocol: 'ws'
+      protocol: protocol === 'wss' ? 'wss' : protocol
```

Comment:
sorry, I guess it should be: `protocol === 'wss' || protocol === 'ws') ? protocol : 'ws',` goal here is to not change ws/wss, but to convert `http -> ws`, `https -> wss`